### ISSUE TITLE
@link and @see tags ignored (inline and regular)

### ISSUE THREAD

**Issue by lkrms** [ApiGen/ApiGen#1094]
I've been looking for a PhpDocumentor alternative with generics support (or even just graceful degradation when PHPDoc syntax isn't recognised), and ApiGen is leading the pack. Kudos on all the hard work getting it to this point!

Aside from ignored code blocks in `@param` descriptions (which may or may not be deliberate on ApiGen's part), the only issue I have is the lack of `@see` (and, less importantly, `@link`) processing. I use inline `@see` for cross-referencing quite extensively.

Is this just a v7 completeness thing, or is there a bigger picture I'm missing? Would there be any interest in a PR to get them working?

**Comment by JanTvrdik** [comment]
> Is this just a v7 completeness thing?

Yes. It's something I'd like add soonish (in the next ~2 weeks), its one of few blocking features for v7.0 stable release.

> Would there be any interest in a PR to get them working?

No. I have already started working on it a bit. I just need to finish it.
